### DISCUSSION TITLE
Opentelemetry javaagent with jetty-jakarta-servlet

### DISCUSSION THREAD

**Discussion by andythsu** [open-telemetry/opentelemetry-java-instrumentation#10400] [Q&A]
Hello,
I have two services, one is simply a jetty proxy server and the other is my application. I have started my proxy with javaagent, and it's able to log the HTTP requests from jetty server. However, after I rewrite the target, it doesn't seem to export the traces from the other application.
For example, when I hit `http://localhost:8080`, it will have a trace called `POST /*`
However, later the endpoint will be re-written, but javaagent won't combine any traces afterwards to this `POST /*` trace. 

To be more specific, here's my traces

![image](https://github.com/open-telemetry/community/assets/18670495/5f74b4b3-b739-4bf7-8cf1-a7a5bc9dd15a)

There should be a trace in between `preConnectionHook` and `postConnectionHook` since the request's endpoint gets re-written and executed. But traces are not being logged.

Any idea is appreciated!

**Comment by laurit** [comment]
Please provide a minimal application that reproduces your issue along with any necessary instructions.

**Reply by andythsu** [reply]
issue has been solved. It's because of an error in the code. Thanks!
